org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -8,6 +8,8 @@
  */
 package org.openhab.binding.zigbee.handler;
 
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.*;
```

Comment:
Personally I don't like static imports - especially importing all constants in this way as I think it's prone to error. Is this something DT do as standard? My preference is to define the class name with the constants - I know it's arguably less readable, but IMHO it's more understandable, and less prone to error.

Reply:
Not a standard at DT :) I, personally, am fine with both, as long as it is consistent within the class... So if you prefer not using static imports, fine with me, Chris.

Reply:
OK. I will remove the static imports and use the class name instead.